src/Core/Components/Nav/FluentNav.razor.cs

```diff
@@ -0,0 +1,257 @@
+/// <list type="bullet">
+/// <item><description><see cref="FluentNavItem"/> - A simple navigation item. When used as a sub item in a <see cref="FluentNavCategory"/>, no icon will be rendered</description></item>
+/// <item><description><see cref="FluentNavCategory"/> - A grouped set of navigation items</description></item>
+/// <item><description><see cref="FluentNavSectionHeader"/> - A section header to organize navigation</description></item>
+/// <item><description><see cref="FluentNavDivider"/> - A visual divider between sections</description></item>
+/// </list>
+/// </remarks>
+public partial class FluentNav : FluentComponentBase
+{
+    private const string JAVASCRIPT_FILE = FluentJSModule.JAVASCRIPT_ROOT + "Nav/FluentNav.razor.js";
+    internal bool _navOpen = true;
+    private readonly List<FluentNavCategory> _categories = [];
+    private bool _previousUseSingleExpanded;
+
+    /// <summary />
+    public FluentNav(LibraryConfiguration configuration) : base(configuration)
+    {
+        Id = Identifier.NewId();
+        _previousUseSingleExpanded = UseSingleExpanded;
+    }
+
+    /// <summary />
+    protected string? ClassValue => DefaultClassBuilder
+        .AddClass("fluent-nav")
+        .Build();
+
+    /// <summary />
+    protected string? StyleValue => DefaultStyleBuilder
+        .Build();
+
+    /// <summary>
+    /// Gets or sets the icon to display besides the app title.
+    /// </summary>
+    [Parameter]
+    public Icon? AppIcon { get; set; }
+
+    /// <summary>
+    /// Gets or sets the title to display at the top of the menu.
+    /// </summary>
+    [Parameter]
+    public string? AppTitle { get; set; }
+
+    /// <summary>
+    /// Gets or sets the link to use for the app item.
+    /// Defaults to homepage ("/").
+    /// </summary>
+    [Parameter]
+    public string AppLink { get; set; } = "/";
+
+    /// <summary>
+    /// Gets or sets whether to use the header with the hamburger icon.
+    /// Defaults to false until we have a good way to make this work with the LayoutHamburger component.
+    /// </summary>
+    [Parameter]
+    public bool UseHeader { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the icon to display for collapsing/expanding the nav menu.
+    /// By default, this icon is a hamburger icon.
+    /// </summary>
+    [Parameter]
+    public Icon ToggleIcon { get; set; } = new CoreIcons.Regular.Size20.LineHorizontal3();
+
+    /// <summary>
+    /// Gets or sets whether to enable using icons in the nav items.
+    /// </summary>
+    [Parameter]
+    public bool UseIcons { get; set; } = true;
+
+    /// <summary>
+    /// Gets or sets whether to allow just one expanded category or multiple
+    /// </summary>
+    [Parameter]
+    public bool UseSingleExpanded { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the density of the nav menu item.
+    /// </summary>
+    [Parameter]
+    public NavDensity? Density { get; set; }
```

Comment:
The `Density` is a Nav parameter, so the `density` HTML attribute must be on this `.fluent-nav` element (and not on ALL **navitems**. The single difference will be an update in the CSS.